### PR TITLE
[NO-TASK]: Railgun signature verify

### PR DESCRIPTION
**Pull request by Danswar**
#### Pre-Release

- [ ] Check migrations
  - No database related infos (sqldb-xxx)
  - Impact on GS (new/removed columns)
- [ ] Check for linter errors (in PR)
- [ ] Test basic user operations (on [DFX services](https://dev.services.dfx.swiss))
  - Login/logout
  - Buy/sell payment request
  - KYC page

#### Post-Release

- Test basic user operations
- Monitor application insights log

### CODE REVIEW

File: src/integration/railgun/railgun.module.ts
Comment (davidleomay) [inline src/integration/railgun/railgun.module.ts:1]: Move Railgun folder into blockchain folder?

File: src/config/config.ts [hunk 1]
```diff
@@ -104,8 +104,9 @@ export class Configuration {
   cardanoAddressFormat = 'stake[a-z0-9]{54}';
   defichainAddressFormat =
     this.environment === Environment.PRD ? '8\\w{33}|d\\w{33}|d\\w{41}' : '[78]\\w{33}|[td]\\w{33}|[td]\\w{41}';
+  railgunAddressFormat = '0zk[a-z0-9]{1,124}';
```
Comment (davidleomay) [inline src/config/config.ts:108]: Signature format is the same as Ethereum?

Comment (Danswar) [reply src/config/config.ts:108]: no, it is not, this one starts with **0zk** and the length can be up to 128 long

Comment (Danswar) [reply src/config/config.ts:108]: Please forget the previous answer. Yes, signature is similar than Ethereum but it needs a different algorithm for verification. Addresses are different.

Comment (davidleomay) [reply src/config/config.ts:108]: Ok, interesting 👍

File: src/integration/railgun/railgun.module.ts
Comment (davidleomay) [reply src/integration/railgun/railgun.module.ts:1]: Hm yes, funny thing 🤔 let's leave it as is 👍

**Review by davidleomay — APPROVED** [review]

**Pull request merged**